# 03-no-append-domain-localhost.patch by Luigi Gangitano <luigi@debian.org>
#
# Don't apply "append_domain" setting to "localhost" in unqualified URLs

--- src/url.c	2002-09-12 07:21:00.000000000 +0200
+++ src/url.c	2002-10-10 19:37:43.000000000 +0200
@@ -313,5 +313,5 @@
     }
 #endif
-    if (Config.appendDomain && !strchr(host, '.'))
+    if (Config.appendDomain && !strchr(host, '.')&& strcasecmp(host,"localhost") != 0)
 	strncat(host, Config.appendDomain, SQUIDHOSTNAMELEN);
     /* remove trailing dots from hostnames */
